linkerd/docs/load_balancer.md

```diff
@@ -10,44 +12,65 @@ routers:
       decayTimeMs: 15000
 ```
 
-Specifies a load balancer to use.  It must be an object containing keys:
-
-  * *kind* -- One of the supported load balancers.
-  * *enableProbation* -- Optional.  Controls whether endpoints are eagerly evicted from
-    service discovery. (default: true)
-    See Finagle's [LoadBalancerFactory.EnableProbation](https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/LoadBalancerFactory.scala#L28)
-  * Any options specific to the load balancer.
+<aside class="notice">
+These parameters are available to the loadbalancer regardless of kind. The loadbalancer may also have kind-specific parameters.
+</aside>
 
-If unspecified, p2c is used. Current load balancers include:
+Key | Default Value | Value Description
+--- | ------------- | -----------------
+kind | `p2c` | `p2c`, `ewma`, `aperture`, or `heap`
+enableProbation | `true` | If `true`, endpoints are eagerly evicted from service discovery. See Finagle's [LoadBalancerFactory.EnableProbation](https://github.com/twitter/finagle/blob/develop/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/LoadBalancerFactory.scala#L28)
 
 [p2c]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-least-loaded
 [ewma]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-peak-ewma
 [aperture]: https://twitter.github.io/finagle/guide/Clients.html#aperture-least-loaded
 [heap]: https://twitter.github.io/finagle/guide/Clients.html#heap-least-loaded
 
-## [p2c][p2c]
+## Power of Two Choices: Least Loaded
+
+kind: `p2c`
+
+<aside class="success">
+  Learn more about p2c and how to configure it via <a target="_blank" href="https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-least-loaded">finagle's documentation</a>
+</aside>
+
+Key | Default Value | Value Description
+--- | ------------- | -----------------
+maxEffort | `5` | The number of times a load balancer can retry if the previously picked node was marked unavailable.
+
+## Power of Two Choices: Peak EWMA
 
-p2c supports the following options (see [here][p2c] for option semantics and defaults):
+kind: `ewma`
 
-* *maxEffort* -- Optional.
+<aside class="success">
+  Learn more about ewma and how to configure it via <a target="_blank" href="https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-peak-ewma">finagle's documentation</a>
```

Comment:
If there are, this is my first time hearing about it. `target="_blank"` is standard 2016 website practice when linkifying external urls.